warehouse/query-core/src/main/java/datawave/query/attributes/UniqueGranularity.java

```diff
@@ -90,17 +130,28 @@ private static class DateTimeValueFormatter implements Function<String,String> {
         
         private static final Logger log = Logger.getLogger(DateTimeValueFormatter.class);
         private final SimpleDateFormat formatter;
+        private boolean isTenth = false;
         
         private DateTimeValueFormatter(String pattern) {
             this.formatter = new SimpleDateFormat(pattern);
         }
         
+        private DateTimeValueFormatter(String pattern, boolean isTenth) {
+            this.formatter = new SimpleDateFormat(pattern);
+            this.isTenth = isTenth;
+        }
+        
         @Override
         public String apply(String value) {
             try {
                 // Attempt to format the denormalized date value.
                 Date date = DateNormalizer.DATE_NORMALIZER.denormalize(value);
-                return formatter.format(date);
+                String formattedDate = formatter.format(date);
+                if (!isTenth) {
+                    return formatter.format(date);
+                } else {
+                    return formattedDate.substring(0, 1);
```

Comment:
The tenth thing isn't clicking for me yet. It looks like if my date was `2022-01-01T12:30:00`, then this function would just return `3`. Off the top of my head, I'd think this should be `2022-01-01T12:3`

Reply:
When I talked to the ticket creator last week I believe they were only interested in the tenths digit of the minutes column exclusively, but I'll circle back around an confirm this.

Reply:
Update: Got clarification that your format is what I should change it to, so I'll update this on next commit